### ISSUE TITLE
indicator of history when there is none

### ISSUE THREAD

**Issue by geekgonecrazy** [RocketChat/Rocket.Chat#787]
An empty private group / channel is created with all users added at creation you see this:
![image](https://cloud.githubusercontent.com/assets/51996/9838089/9818d178-5a19-11e5-9c75-45a4d57d4579.png)

I'd think it'd show "Start of conversation" like it does in direct messages.

**Comment by AdrianoCahete** [comment]
The "has more" text don't follow user's language too (at least not in portuguese).

**Comment by geekgonecrazy** [comment]
Ah yes so also apparently needs switched to i18n or at least have it updated.

**Comment by graywolf336** [comment]
To give a little more detail:

```
I20150913-14:45:23.109(-5)? [methods] canAccessRoom ->  userId: 7zNDYqLZPE7RRCoCa rid: RHCPdEAJzG4vJAZXD
I20150913-14:45:23.112(-5)? Exception while invoking method 'loadHistory' TypeError: Cannot read property 'ts' of undefined
I20150913-14:45:23.113(-5)?     at [object Object].Meteor.methods.loadHistory (server\methods/loadHistory.coffee:28:18)
I20150913-14:45:23.113(-5)?     at [object Object].methodMap.(anonymous function) (packages/meteorhacks:kadira/lib/hijack/wrap_session.js:164:1)
I20150913-14:45:23.113(-5)?     at maybeAuditArgumentChecks (packages/ddp/livedata_server.js:1617:1)
I20150913-14:45:23.113(-5)?     at packages/ddp/livedata_server.js:648:1
I20150913-14:45:23.113(-5)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20150913-14:45:23.113(-5)?     at packages/ddp/livedata_server.js:647:1
I20150913-14:45:23.113(-5)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20150913-14:45:23.114(-5)?     at [object Object]._.extend.protocol_handlers.method (packages/ddp/livedata_server.js:646:1)
I20150913-14:45:23.114(-5)?     at packages/ddp/livedata_server.js:546:1
```

Is what I get when creating a new channel.

**Comment by rodrigok** [comment]
Seems that this was fixed by #793
